src/midi/midi_mt32.cpp

```diff
@@ -204,6 +204,20 @@ static void init_mt32_dosbox_settings(Section_prop &sec_prop)
 	        "  <custom>:  Custom filter definition; see 'sb_filter' for details.");
 }
 
+static void add_mt32_dosbox_messages()
+{
+	MSG_Add("MT32_NO_SUPPORTED_MODELS", "No supported models present.");
+
+	MSG_Add("MT32_ROM_NOT_LOADED", "No ROM is currently loaded");
+
+	MSG_Add("MT32_INVENTORY_TABLE_MISSING_LETTER", "-");
```

Comment:
Do we need this?

Reply:
Ideally we'd use some universally understandable icons for these, like Ikea uses in their text-free instruction sheets :smile: 

I think it makes sense, because the dash may not translate to a mere placeholder or 'nothing' in other languages.

![2022-09-09_07-22](https://user-images.githubusercontent.com/1557255/189372466-1b41ff50-36c2-4df7-ac0d-874d7cbaf9b7.png)

https://www.pinhok.com/kb/russian/337/russian-punctuation-marks/

Maybe on other languages, they have more appropriate characters for 'nothing/none/empty', that were common in the DOS era.  

This reminds me of the the Lucas Arts installer using CP850, where is used poorly selected 'checkbox' symbols that @Burrito78 flagged.

Reply:
Interesting, it's better to just add it then. It's a small thing anyway it gives us more flexibility, agreed.